graylog2-server/src/main/java/org/graylog2/plugin/PluginModule.java

```diff
@@ -254,6 +255,14 @@ protected void addEventModifier(Class<? extends EventModifier> eventModifierClas
         eventModifierBinder().addBinding().to(eventModifierClass);
     }
 
+    protected Multibinder<EventQuerySearchTypeSupplier> eventQuerySearchTypeSupplierBinder() {
```

Comment:
We need to call this in a module to ensure the binder exists. Otherwise, we will get errors when no suppliers are bound.

Reply:
Aye, added in https://github.com/Graylog2/graylog2-server/pull/18372/files#diff-db1276a85077c687cc2220b4ef171a8f7cc78d02b218db30da56c6e98bab7957R191